### ISSUE TITLE
Loot journal entry items with null quantity defined disappear when any other quantity value is changed.

### ISSUE THREAD

**Issue by madcatprimary** [ironmonk88/monks-enhanced-journal#241]
In a loot journal, items added to the page that have no quantity displayed will disappear if the quantity of any other item is changed. If multiple items have no quantity, they will all disappear forever, simultaneously. This also happens if any item in the list is deleted. Doing a deletion to any item on the page deletes all items with no quantity value assigned. Perhaps the quantity value should default to 1. 

Making sure to specify the value of each item before adding anymore does prevent these disappearances.

**Comment by ironmonk88** [comment]
That's working pretty much as intended.  Loot sheet isn't a Shop, so if there's no quantity it gets deleted.
That being said... how are you adding things with zero quantity?  When I drag anything into the Loot sheet it prompts for a quantity to add, and if it's set to 0 it won't be added.  And adding items from a roll table should default to 1.

**Comment by madcatprimary** [comment]
I was dragging items from an actor that I have been using as a loot container into a loot journal entry. Items that had a quantity of > 1 showed with their quantity, but most of the single items were added with a null value for the quantity. When doing this, there's no dialog to set quantity, which is fine since there should already be a quantity in the actor entry to use. I haven't looked at it in a few weeks, but trying to duplicate it now isn't working. The specific items that were on the actor before have since been sold off, so they aren't there to try to recreate this. In the meantime, I have avoided using the loot journals since the items I put on it were lost to the ether, forcing me to try to recreate them from memory and I didn't want to have to deal with that happening again. I'll try using it some more and get back to you if I can make a reproducible set of steps.
